modules/nw-metallb-configure-address-pool.adoc

```diff
@@ -0,0 +1,53 @@
+[id="nw-metallb-configure-address-pool_{context}"]
+= Configuring an address pool
+
+Add a cluster administrator, you can add address pools to your cluster to control the IP addresses that MetaLLB can assign to load-balancer services.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a file, such as `addresspool.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  namespace: metallb-system
+  name: doc-example
+spec:
+  protocol: layer2
+  addresses:
+  - 203.0.113.1-203.0.113.10
+  - 203.0.113.65-203.0.113.75
+----
+
+. Apply the configuration for the address pool:
++
+[source,terminal]
+----
+$ oc apply -f addresspool.yaml
+----
+
+.Verification
+
+* View the address pools:
++
+[source,terminal]
+----
+$ oc get -n metallb-system addresspool
```

Comment:
@mikemckiernan Could we add 'oc describe addresspool <address pool name> -n metallb-system' in case user needs to know the ranges configured for pool? Or it is assumed describe can also be used.